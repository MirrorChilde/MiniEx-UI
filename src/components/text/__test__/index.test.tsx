/**
 * @jest-environment jsdom
 */

import React from 'react';
import { Text } from '..';
import { render } from '@testing-library/react';

describe('Text', () => {
  it('default text', () => {
    const { container } = render(<Text title='123' />);
    // console.log(container);
    expect(container.firstChild).toMatchSnapshot();
  });
});
