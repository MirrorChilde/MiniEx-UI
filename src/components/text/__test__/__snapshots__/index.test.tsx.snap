// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text default text 1`] = `
<h1>
  title:
  123
</h1>
`;
